lib/pages/setting/setting.dart

```diff
@@ -66,13 +106,25 @@ class _SettingPageState extends State<SettingPage> {
                     onChanged: onSwitchPressed,
                   ),
                 ),
-                const SettingListItem(
+                SettingListItem(
                   title: '시간설정',
                   icon: Icons.timer,
                   destination: '',
-                  widget: Text(
-                    '00:00',
-                    style: TextStyle(fontSize: 15, fontWeight: FontWeight.w600),
+                  widget: TextButton(
+                    onPressed: switchValue == false ? null : onSetTime,
```

Comment:
저번에 이거 놓쳤군.. hasSelectedTime 이런 네이밍은 어떠신가여? switchValue 함수같아서 ㅎㅎ + 가급적이면 삼항쓸 때 조건이 truthy하게 쓰면 어떨까 그게 좀 더 잘 읽히는 것 같음!!!

```suggestion
                    onPressed: hasSelectedTime == true ? onSetTime: null,
```

근데 생각해보니까 time을 optional로 두고 time 값 있는지로 체크하게 되면 변수 하나로만 해도 될듯? text 넘겨줄 때 값 없으면 format.parse(time) 요걸로 default 처리하구

Reply:
넴 !@ 네이밍 더 신경써야지 🫠
time을 format.parse(time)으로 처리하면 disabled할 때마다 현재 시각으로 바뀌는데 그걸 말하는건가용 ?!

Reply:
@narinn-star 
초깃값이 `DateFormat.jm().format(DateTime.now())`이걸로 되어있잔아 그래서 time 초깃값을 없애고(nullable) time이 값이 있으면 hasSelectedTime==true인거랑 같다고 생각햇어여 아닌가용